ios/keyman/Keyman/Keyman/SetUpViewController/SetUpViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  SetUpViewController.swift
+//  Keyman
+//
+//  Created by Gabriel Wong on 2017-09-05.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+import UIKit
+
+// TODO: Refactor common functionality from InfoViewController
```

Comment:
I don't believe I see this comment in the originals.  Was this in reference to something meant for this PR?

Reply:
That's for later refactoring after this PR is merged.